docker-compose/docker-compose.yml

```diff
@@ -87,15 +85,50 @@ services:
 
   seq:
     image: datalust/seq:latest
+    container_name: seq
     ports:
-      - "8081:80"
+      - "8082:80"
       - "5341:5341"
     volumes:
       - seq-data:/data
     environment:
       - ACCEPT_EULA=Y
     restart: unless-stopped
 
+  adminApi:
+    image: nmshd-adminapi
+    container_name: adminApi
+    hostname: adminApi
+    build:
+      context: ..
+      dockerfile: AdminUi/src/AdminUi/Dockerfile.debug
+    ports:
+      - "5173:5000"
+    depends_on:
+      # - ms-sql-server
+      - rabbitmq
+      - postgres
+      - seq
```

Comment:
If the `adminApi` depends_on `seq`, shouldn't the `consumerApi` follow suit?

Reply:
good catch, ty